sys/include/net/loramac.h

```diff
@@ -105,8 +107,9 @@ extern "C" {
 /**
  * @brief Default LoRaWAN region
  */
-#if !defined(LORAMAC_REGION_EU_868) && !defined(LORAMAC_REGION_IN_865)
-#define LORAMAC_REGION_EU_868                  (1)
+#if !IS_ACTIVE(CONFIG_LORAMAC_REGION_EU_868) \
+    && !IS_ACTIVE(CONFIG_LORAMAC_REGION_IN_865)
+#define CONFIG_LORAMAC_REGION_EU_868           1
```

Comment:
Removing the parenthesis should be done in the first commit (130519ade5e157e2437554a279410c4da2c10eaf).

Rationale: This is the first commit that will build (all the others fail because of the extra parenthesis). If someone does `git bisect` and hit those commits, the binary won't compile. Thus, all commits should compile.

Reply:
I have used `#Ifdef `for other commits. Just checked. It seems to build just fine. Not sure if I am missing something here